<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="titleBarWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#titleBarWidget{background:white;}</string>
     </property>
     <layout class="QGridLayout" name="titleBarWidgetGridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="titleBarLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="iconButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:0px;
background:transparent;margin:5px 0px 0px 0px;}
QPushButton:hover{border:0px;
background:transparent;margin:5px 0px 0px 0px;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icon.ico</normaloff>../icon.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">margin:5px 0px 0px 5px;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="titleBarLayoutHSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="helpButton">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#e2e2e2;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../style/style.qrc">
            <normaloff>:/custom/icons/help.png</normaloff>:/custom/icons/help.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingButton">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#e2e2e2;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../style/style.qrc">
            <normaloff>:/custom/icons/settings.png</normaloff>:/custom/icons/settings.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="minimizeButton">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>最小化</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#e2e2e2;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../style/style.qrc">
            <normaloff>:/custom/icons/minimize.png</normaloff>:/custom/icons/minimize.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="maximizeButton">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>最大化</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#e2e2e2;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../style/style.qrc">
            <normaloff>:/custom/icons/maximize.png</normaloff>:/custom/icons/maximize.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>关闭</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:0px;
background:transparent;}
QPushButton:hover{border:0px;
background:transparent;background-color:#fb7373;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../style/style.qrc">
            <normaloff>:/custom/icons/close.png</normaloff>:/custom/icons/close.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#mainWidget{background:white;}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="ipEdit">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>监听地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEdit">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>监听端口号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dirEdit">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>主目录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;}
QPushButton:hover{background:#d2d2d2;}
QPushButton:press{background:#d2d2d2;}
QPushButton{background:#07c160}
QPushButton:hover{background-color: rgb(56, 205, 127);}
QPushButton:disabled{background-color: rgba(56, 205, 127,50%);}</string>
          </property>
          <property name="text">
           <string>开始监听</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="endButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;background:#fb7373}
QPushButton:hover{background:#d2d2d2;background:rgba(251, 115, 115,80%);}
QPushButton:press{background:#d2d2d2;background:rgba(251, 115, 115,80%);}
QPushButton:disabled{background:rgba(251, 115, 115,50%);}</string>
          </property>
          <property name="text">
           <string>停止监听</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="indexButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;}
QPushButton:hover{background:#d2d2d2;}
QPushButton:press{background:#d2d2d2;}</string>
          </property>
          <property name="text">
           <string>访问首页</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;}
QPushButton:hover{background:#d2d2d2;}
QPushButton:press{background:#d2d2d2;}</string>
          </property>
          <property name="text">
           <string>日志窗口</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearLogButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;}
QPushButton:hover{background:#d2d2d2;}
QPushButton:press{background:#d2d2d2;}</string>
          </property>
          <property name="text">
           <string>清空日志</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="directoryButton">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{color:black;background:#e9e9e9;border:0px solid rgb(167, 166, 170);border-radius:5px;padding:0px 5px 0px 5px;}
QPushButton:hover{background:#d2d2d2;}
QPushButton:press{background:#d2d2d2;}
QPushButton:disabled{color:gray;}</string>
          </property>
          <property name="text">
           <string>打开主目录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="styleSheet">
         <string notr="true">QSplitter:handle{background:transparent;height:2px;}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QTableWidget" name="requestTableWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
        </widget>
        <widget class="QTextEdit" name="logEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:0px;</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../style/style.qrc"/>
 </resources>
 <connections/>
</ui>
